app/helpers/passport.js

```diff
@@ -1,17 +1,20 @@
 'use strict';
+
+var request = require('request');
 var GitHubStrategy = require('passport-github').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
 
 var User = require('../models/user');
+var Account = require('../models/account');
 var config = require('../../config');
 
-// expose this function to our app using module.exports
 module.exports = function(passport) {
     passport.serializeUser(function(user, done) {
```

Comment:
No I know but why do we need to export it? Do you need it at the route mounting callsite?